docs/earthfile/earthfile.md

```diff
@@ -677,6 +677,14 @@ SAVE IMAGE my-example-registry.com/another-image:latest
 ```
 {% endhint %}
 
+{% hint style='danger' %}
+##### Important
+
+Since the introduction of the `--referenced-save-only` feature flag, images are only saved if they are connected to the initial target through a chain of `BUILD` commands. [See the `--referenced-save-only` flag documentation for more details](./features.md#referenced-save-only).
+
+This is automatically enabled when setting [`VERSION 0.6`](#version) in your `Earthfile`.
+{% endhint %}
```

Comment:
Note that this applies to `SAVE ARTIFACT ... AS LOCAL ...` too. Should there be a similar warning there?

Reply:
Yes, probably